staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -2556,7 +2556,10 @@ func (i *IngressDescriber) describeBackendV1(ns string, backend *networkingv1.In
 	}
 	if backend.Resource != nil {
 		ic := backend.Resource
-		return fmt.Sprintf("APIGroup: %v, Kind: %v, Name: %v", *ic.APIGroup, ic.Kind, ic.Name)
+		if ic.APIGroup != nil {
```

Comment:
This is one of those things that's meh no matter what you do.

I'd probably do something like this and only have one return.

```
var apiGroup string
if ic.APIGroup != nil {
  apiGroup = fmt.Sprintf("%v", *ic.APIGroup)
}
```

Reply:
Gotcha